### PR TITLE
Twitter and series

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/lib/getClaimIds.js [hunk 1]
```diff
@@ -21,3 +29,14 @@ export default function getClaimIds(entity, propId) {
     )
     .filter((c) => c); //filter out 'No value' and 'Unknown'
 }
+
+//checks whether the claim has a qualifier of series ordinal, return true if the first one has it
+export function checkIfClaimsHasSeriesOrdinal(entity, propId) {
+  const claims = entity.claims[propId] || [];
+  if (claims.length) {
+    if (getSeriesOrdinal(claims[0])) {
```
Comment (ogroppo) [inline src/lib/getClaimIds.js:37]: are u sure that if the first one has it than it's true for all of them?

Comment (mshd) [reply src/lib/getClaimIds.js:37]: no, but it's sufficient

File: src/lib/addSecondLabel.js [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
   const { labels, simpleClaims } = entity;
   if (!propCode) return;
-  if (propCode === NICKNAME_ID) {
-    const nickname = getSimpleClaimValue(simpleClaims, NICKNAME_ID);
-    if (nickname) {
-      entity.secondLabel = nickname;
+  if ([BIRTH_NAME_ID, NICKNAME_ID, NAME_IN_KANA_ID].includes(propCode)) {
+    const secondLabel = getSimpleClaimValue(simpleClaims, propCode);
+    if (secondLabel) {
+      entity.secondLabel = secondLabel;
```
Comment (ogroppo) [inline src/lib/addSecondLabel.js:15]: Very nice

File: src/constants/properties.js [hunk 3]
```diff
@@ -19,6 +19,7 @@ export const STUDENT_OF_ID = "P1066";
 export const STUDENT_ID = "P802";
 export const LOCATED_IN_ID = "P131";
 export const NICKNAME_ID = "P1449";
+export const NAME_IN_KANA_ID = "P1814";
```
Comment (ogroppo) [inline src/constants/properties.js:22]: what the heck is this

**Review by ogroppo — APPROVED** [review]

File: src/constants/properties.js (hunk 3 above)
Comment (mshd) [reply src/constants/properties.js:22]: Haha :) it's Korean transliteration

**Pull request merged**